cpp/tests/io/parquet_chunked_reader_test.cu

```diff
@@ -1544,7 +1544,8 @@ TEST_F(ParquetChunkedReaderTest, TestNumRowsPerSource)
 
   // Chunked-read rows_to_read rows skipping rows_to_skip from single data source
   {
-    auto const rows_to_skip          = 1'237;
```

Comment:
Were these tests segfaulting before this change?

Reply:
Surprisingly no. The segfaults particularly show up if we have at least one string column in which case the chunking across subpass and output chunks goes haywire. Changed this as we will certainly get a `CUDF_FAIL()` after this PR.